### ISSUE TITLE
cargo bench doesn't report data for Halo2

### ISSUE THREAD

**Issue by zhenfeizhang** [anoma/zkp-compiler-shootout#15]

**Comment by mariari** [comment]
Yeah I need to hook up the current `Halo2` code to the bench-mark

It should be possible to get both backends on the benchmark

**Comment by mariari** [comment]
With the latest commits the current `Halo2` code should be shown in the results
